server/app.py

```diff
@@ -1603,3 +1610,79 @@ def close_rdo(subscription_id):
     response = RDO.update([rdo], update_details)
     app.logger.info(response)
     return rdo
+
+
+# The rdo_converter command finds existing recurring donations that are only tracked in salesforce
+# and converts them to stripe subscriptions. Stripe then passes the new subscription id back to
+# salesforce so that the two systems are in sync. Any recurring donations that are converted will
+# no longer run on the nightly load and will be handled by stripe's scheduled charges instead.
+# Finally, a recurring donation's next charge date is used as the next date that stripe should attempt
+# charging the donor, so no charge occurs at time of conversion.
+@app.cli.command('rdo_converter')
+@click.option('-l', '--limit', default=0,
+              help="The number of initial donation records to look at for conversion. Good for testing small batches.")
+@click.option('-t', '--donation-type', 'donation_type', default=None, 
+            help="The type of donation to convert. Can be one of 'Recurring Donation', 'Business Membership', 'Giving Circle' or 'The Blast'.") 
+@click.option('-s', '--only-staff', 'only_staff', is_flag=True, default=False,
+              help="A flag to limit conversions to recurring donations given by current staff.")
+@click.option('-e', '--email', default=None,
+              help="The particular email address used for some amount of recurring donations. Use this if you only need to convert one persons donations.")
+def rdo_converter(limit, donation_type, only_staff, email):
+    if not donation_type and not email:
+        return "A donation type or email must be passed"
+
+    # If any of the vars passed here are null, they're ignored and the query
+    # is built without them
+    rdos = RDO.list(limit=limit, donation_type=donation_type, email=email)
+    staff_names = []
+
+    # is there a better way to get a list of staff?
+    if only_staff:
+        response = requests.get('https://www.texastribune.org/api/v2/authors/?staff=true&limit=100')
+        staff_results = response.json()['results']
+        staff_names = [staff["name"] for staff in staff_results]
```

Comment:
Yeah, I'll chat with @ashley-hebler, but it might make sense to update this to grab any recurring donations with @texastribune.org emails affiliated as a next step. The only way we'd be able to go off of non-tribune related emails tied to staff is if we got a list of names from membership, and that might be more work all-around than it's worth.

Reply:
Following along! It'd be worth asking if Kassie has that info easily accessible. In the end, if we don't find every staff member currently managed by salesforce, that's not the end of the world. We're going to move everyone eventually anyway.

Reply:
@matthewdylan lets just stick to tt.org to make our lives eaiser!